frontend/src/components/filters/FiltersInfoTS.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { FILTER_TYPE } from "../../constants";
+
+
+const FiltersInfos = ({
+    filters,
+}) => {
+    const appliedFilters = Object.keys(filters).filter((key: any) => filters[key]?.value)
+    const getValue = (key: any) => {
+        const filterValue = filters[key].value
+        const description: any = filters[key].description
+        let valueJSX: any;
+        const descriptionJSX: any = (
+            <>
+                {
+                    <strong>{description.label}: </strong>
+                }
+            </>
+        );
+        const valuesArray = [].concat(filterValue)
+        let labels: any = [];
+        let value = ""
+        switch (description.type) {
+            case FILTER_TYPE.SELECT:
```

Comment:
I get a big linter error here `Unexpected lexical declaration in case block`. It is complaining about declaring local variables in the case statement. The fix is to put the case handling code in `{}` brackets, eg.
```
case THING: {
  const someVar = ...
  break;
}
```

<img width="698" alt="Screen Shot 2023-04-25 at 2 07 13 PM" src="https://user-images.githubusercontent.com/104078019/234364795-3ba7d73f-1c87-49f6-a175-31f1b084ddd3.png">

If you're not getting that error I can help you get the linter set up.